### DISCUSSION TITLE
Constant White Noise

### DISCUSSION THREAD

**Discussion by Vilhelm16** [GrandOrgue/grandorgue#1936] [Help] [answered]
I'm getting white noise constantly. Doesn't matter which sample set. It's not motor noises. It gets worse the more stops I have selected, often whether or not I play anything. It does get worse when I do play. I'm at wit's end as to what it could be - it doesn't match any existing issues. I have completely reinstalled Grandorgue, and the issue continued. Am on Windows.

I have attached what it sounds like. The sample set is Friesach in this case.
[go issue audio.zip](https://github.com/user-attachments/files/16113928/go.issue.audio.zip)

Any ideas?

**Comment by oleg68** [comment]
1. What version of GrandOrgue do you use?
2. What audio interface do you use?
3. Try to record something with GrandOrgue Panel->Recorder/Player->Audio Recorder. Is the noise present in the recorded file or not?
4. Post here the screenshots of File->Settings->Audio and File->Settings->Parameters.

**Reply by Vilhelm16** [reply]
1. Version 3.14.2-1
2. Just using computer default audio devices.
3. Yes, it is present. The audio recording in the original post is from the native audio recorder.
4. By parameters, do you mean options? I've included screenshots of that and the audio settings.
![go screenshot 1](https://github.com/GrandOrgue/grandorgue/assets/27139210/734701fc-ec99-4bfa-9645-54186efe0c47)
![go screenshot 2](https://github.com/GrandOrgue/grandorgue/assets/27139210/2d6a4532-927b-440b-b1e7-29460f6e5e5c)

**Reply by larspalo** [reply, marked as answer]
Oh that should be an easy fix. Just increase the Sample size from 8 bits to at least 16 bits, but 24 could be better (especially if the samples have that bit depth).

**Reply by oleg68** [reply]
> 3\. The audio recording in the original post is from the native audio recorder.

But the internal recorder of GrandOrgue produces .vaw files, not .mp3.

**Reply by Vilhelm16** [reply]
> > 3. The audio recording in the original post is from the native audio recorder.
> 
> But the internal recorder of GrandOrgue produces .vaw files, not .mp3.

So it is. I trimmed this down from something longer, so I guess the editor I trimmed it in converted the file type. Weird.

**Comment by oleg68** [comment]
I agree with @larspalo. There are some steps that may improve the sound quality:
- Increase Sample Size to 16 or even to 24 bits
- If most of samplesets have been recorded in 48000 saples/sec, set the Sample Rate to 48000
- Select `Polyphase` interpolation instead of `Linear` one.

**Comment by oleg68** [comment]
I also noted some growing negative offset in the attached record

![audacity-screen](https://github.com/GrandOrgue/grandorgue/assets/11586506/12e76246-0abd-4663-b4fd-8d0d29dbab11)
